src/api/Compiler.hx

```diff
@@ -179,12 +180,24 @@ class Compiler {
 				return {type:res};
 			}
 
-			var words = [];
+			var words:Array<CompletionItem> = [];
 			
 			if (completionType == CompletionType.DEFAULT)
 			{
 				for( e in xml.nodes.i ){
-					var w = e.att.n;
+					var w:CompletionItem = {n: e.att.n, d: ""};
+					
+					if (e.hasNode.t)
+					{
+						w.t = e.node.t.innerData;
+						w.d = w.t + "</br>";
```

Comment:
Sorry, that's not what I meant :D (I hadn't even figured it was weird)
Whether it is `<br>` `<br/>` or `</br>`, I still think it shouldn't be in the API for SoC's sake :\